File: app/src/main/java/br/com/netflix/model/Movie.java
package br.com.netflix.model;

public class Movie {
    private  int id;
    private String title;
    private String desc;
    private String cast;
    private String coverUrl;

    public Movie(int id, String coverUrl) {
        this.id = id;
        this.coverUrl = coverUrl;
    }

    public Movie(int id, String title, String desc, String cast, String coverUrl) {
        this.id = id;
        this.title = title;
        this.desc = desc;
        this.cast = cast;
        this.coverUrl = coverUrl;
    }

    public int getId() {
        return id;
    }

    public String getTitle() {
        return title;
    }

    public String getDesc() {
        return desc;
    }

    public String getCast() {
        return cast;
    }

    public String getCoverUrl() {
        return coverUrl;
    }
}

File: app/src/main/java/br/com/netflix/model/Movie.kt
package br.com.netflix.model

import com.google.gson.annotations.SerializedName

data class Movie(
    val id: Int,
    @SerializedName("cover_url") val coverUrl: String,
    val title: String,
    val desc: String,
    val cast: String
) {
    constructor(
        id: Int,
        coverUrl: String
    ) : this(id, coverUrl, "", "", "")
}
